packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -227,86 +257,110 @@ const ToggleMenuDrawer = React.forwardRef(
         return (
             <React.Fragment>
                 <a id='dt_mobile_drawer_toggle' onClick={toggleDrawer} className='header__mobile-drawer-toggle'>
-                    <Icon icon='IcHamburger' width='16px' height='16px' className='header__mobile-drawer-icon' />
+                    <Icon
+                        icon={is_dashboard && !is_logged_in ? 'IcHamburgerWhite' : 'IcHamburger'}
+                        width='16px'
+                        height='16px'
+                        className='header__mobile-drawer-icon'
+                    />
                 </a>
                 <MobileDrawer
-                    alignment='left'
+                    alignment={is_dashboard ? 'right' : 'left'}
                     icon_class='header__menu-toggle'
                     is_open={is_open}
                     toggle={toggleDrawer}
                     id='dt_mobile_drawer'
                     enableApp={enableApp}
                     disableApp={disableApp}
-                    title={localize('Menu')}
-                    livechat={<LiveChat is_mobile_drawer />}
+                    title={title || title === '' ? title : localize('Menu')}
+                    livechat={is_dashboard ? null : <LiveChat is_mobile_drawer />}
```

Comment:
I think we should still have Livechat in mobile drawer for dwallet. Maybe it's missed in the design. 
Can you please check with design team?

Reply:
Sure, current responsive design is not finalized yet, so I was thinking once they come up with final decision, I'll add it back if required. Basically this code is not finalized yet